packages/@okta/i18n/src/properties/login.properties

```diff
@@ -840,7 +843,7 @@ oie.numberchallenge.warning = Haven't received a push notification yet? Try open
 ## {0} refers to a number which is the correct answer for the number challenge
 oie.numberchallenge.instruction = On your mobile device, open the Okta Verify prompt, then tap <$1>{0}</$1> in Okta Verify to continue.
 oie.numberchallenge.force.upgrade.title = Update Okta Verify
-oie.numberchallenge.force.upgrade = Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifcations, update Okta Verify to the latest version, then try again.
+idx.authenticator.app.method.push.force.upgrade.number_challenge = Your response was received, but your Okta Verify version is no longer supported by your organization. To verify your identity with push notifications, update Okta Verify to the latest version, then try again.
```

Comment:
This is something I am fixing from my previous PR. We just need to add the server returned i18n key to login.properties to account for i18n.

Reply:
qq does all the strings in error messages get translated at some base level in OIE ?

Reply:
Yes, for messages returned from the API  [i18nTransformer](https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/i18nTransformer.js#L268-L278) will do a look up in login.properties for the `i18n.key` else if will render the English text returned by the server.

Reply:
Backend should return error keys with prefix `oie` instead of `idx`.. We should form an opinion on this with backend teams..

Reply:
I'll check with backend folks on this.

Reply:
did we get an answer on this ? would be good to have the same convention.

Reply:
yes, the keys are updated to oie now